Add valid-url to dependencies.

**Version**
master branch at cde7dce154d44154451a015751a775991e6d987d

**Steps to reproduce**
1. Run `git clone https://github.com/TryxAPI/lrs-conformance-tests.git lrs-conformance-tests`.
2. Run `cd lrs-conformance-tests`.
3. Run `npm install`.
4. Run `npm link`.
5. Run the tests with the options `authUser`, `authPass`, `basicAuth`, and `endpoint`.

**Expected behaviour**
Step 5 should run the tests successfully.

**Actual behaaviour**
Step 5 actually displays an error (see below).

```
module.js:340
    throw err;
          ^
Error: Cannot find module 'valid-url'
    at Function.Module._resolveFilename (module.js:338:15)
    at Function.Module._load (module.js:280:25)
    at Module.require (module.js:364:17)
    at require (module.js:380:17)
    at Object.<anonymous> (c:\Program Files (x86)\Ampps\www\ll\tests2\test\v1_0_2\non_templating.js:3732:156)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
    at Module.require (module.js:364:17)
```

**Additional information**
Required by [test/v1_0_2/non_templating.js:3732:156](https://github.com/TryxAPI/lrs-conformance-tests/blob/master/test/v1_0_2/non_templating.js#L3732)

--- a/package.json
+++ b/package.json
@@ -42,6 +42,7 @@
     "string": "^3.1.0",
     "super-request": "0.0.8",
     "supertest": "^0.13.0",
-    "supertest-as-promised": "^1.0.0"
+    "supertest-as-promised": "^1.0.0",
+    "valid-url": "^1.0.9"
   }
 }

